catalog/solr/catalog-solr-offline-gazetteer/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- /**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/ -->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+
+  <reference id="solrFactory" interface="org.codice.solr.factory.SolrClientFactory"/>
+
+  <bean id="offlineGazetteerPlugin"
+    class="ddf.catalog.solr.offlinegazetteer.CatalogGazetteerForwardingPlugin">
+    <argument ref="solrFactory"/>
+  </bean>
+  <service ref="offlineGazetteerPlugin">
+    <interfaces>
+      <value>ddf.catalog.plugin.PostIngestPlugin</value>
+      <value>ddf.catalog.plugin.PreQueryPlugin</value>
+    </interfaces>
+  </service>
+
+  <bean id="gazetteerQueryOfflineSolr"
+    class="ddf.catalog.solr.offlinegazetteer.GazetteerQueryOfflineSolr">
+    <argument ref="solrFactory"/>
+  </bean>
+  <service ref="gazetteerQueryOfflineSolr"
+    interface="org.codice.ddf.spatial.geocoding.GeoEntryQueryable" ranking="80"/>
```

Comment:
❓ why 80 ranking?

Reply:
That is its service ranking. If there are multiple services providing the same interface, then the service with the highest ranking will win. If there is a service ranking tie then the first bundle to start will win. I think Ryan set the ranking to 80 so it would beat the standard profile spatial geocoding plugin which has the default service ranking of 1.